app/routes/users/components/UserProfile.tsx

```diff
@@ -395,9 +395,9 @@ const UserProfile = (props: Props) => {
             {hasFrame && (
               <Image alt="Golden frame" className={styles.frame} src={frame} />
             )}
-            <ProfilePicture user={user} size={150} />
+            <ProfilePicture user={user} alt="Profile picture" size={150} />
```

Comment:
A `<ProfilePicture />` component shouldn't really need a `alt` prop as it should always be `{user.fullname}'s profile picture` or something.